src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -560,6 +560,81 @@ public bool ReplayGain
             }
         }
 
+        /// <summary>
+        /// Gets or sets the audio pitch.
+        /// </summary>
+        /// <value>The value indicating whether or not AudioPitch is enabled. The default is false.</value>
+        /// <remarks>This function is used for audio content only.</remarks>
+        /// <remarks>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</remarks>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <seealso cref="AudioPitch"/>
+        /// <since_tizen> 6 </since_tizen>
+        public bool AudioPitchEnabled
```

Comment:
I just wonder `AudioPitchEnabled` can be set by user or internal restriction for audio only content?
If it can be set and controlled by user, why is it not just included in `AudioPitch` setter?

Reply:
* @remarks Enabling pitch control could increase the CPU usage on some devices.
i forgot to add this description. i'll update it on the next patch

Reply:
As discussed in offline, `AudioPitchEnable` have state restriction unlike `AudioPitch`, 
so it would be better that separate it.